### DISCUSSION TITLE
My selected school has incorrect or incomplete information e.g. domains or campus location

### DISCUSSION THREAD

**Discussion by fankehuang** [community/community#124315] [GitHub Education]
On GitHub, the student email suffix for Dongguan Polytechnic is @dgpt.edu.cn, but the suffix for the email I obtained from my school is @stu.dgpt.edu.cn, which prevents me from submitting my application correctly. I am from Mainland China and can provide information from the China Higher Education Student Information website and my student ID to prove my identity as a student of this school. As shown in the video, our school has two types of email accounts: student and teacher. I have recorded my login process. We don't have a website that publicly displays the email domain for educational purposes, and as an ordinary student, I am unable to obtain documents stamped by the school. Please understand and support my request to activate GitHub Education.

**Comment by fankehuang** [comment]
I cannot upload videos larger than 10MB. If you have any suggestions, please let me know, and I will do my best to verify the student's identity.

**Reply by java-walker** [reply]
解决了吗

**Reply by java-walker** [reply]
如果需要添加该域名需要学校提供什么文件吗？

**Comment by ayoub-laaouaoucha** [comment]
To address your issue with the GitHub Education application, you can write a detailed explanation to GitHub Support. Here’s a suggested template for your request:

---

Subject: Request for GitHub Education Activation with @stu.dgpt.edu.cn Email Domain

Dear GitHub Education Support Team,

I hope this message finds you well. I am writing to request assistance with activating my GitHub Education benefits. I am a student at Dongguan Polytechnic in Mainland China. According to the information provided by GitHub, the recognized email suffix for students from my institution is `@dgpt.edu.cn`. However, the email address I received from my school has the suffix `@stu.dgpt.edu.cn`, which is causing issues with my application.

Here are the key points I would like to bring to your attention:

1. **Institutional Email Domains**: At Dongguan Polytechnic, there are two types of email accounts:
    - `@dgpt.edu.cn` for teachers and administrative staff.
    - `@stu.dgpt.edu.cn` for students.

2. **Proof of Student Status**:
    - I can provide my student ID and information from the China Higher Education Student Information website to verify my status as a student of Dongguan Polytechnic.
    - I have recorded my login process to my school email account, which demonstrates my affiliation with the institution.

3. **Lack of Public Documentation**:
    - Our institution does not have a publicly accessible website that displays the email domain used for educational purposes.
    - As an ordinary student, I do not have access to documents stamped by the school administration to officially confirm my email domain.

Given these circumstances, I kindly request your understanding and support in activating my GitHub Education benefits using my `@stu.dgpt.edu.cn` email address. I am ready to provide any additional information or documentation necessary to verify my student status.

Thank you for your time and assistance.

Best regards,

[Your Name]  
[Your GitHub Username]  
[Your Email Address]  
[Dongguan Polytechnic Student ID]

---

Make sure to include any necessary attachments, such as screenshots of your email account, your student ID, and any other relevant documentation. This comprehensive approach will help GitHub Support understand your situation and hopefully grant you the GitHub Education benefits.
